### ISSUE TITLE
Bug: Color Picker "saved colors" faint outline is off spec

### ISSUE THREAD

**Issue by bstifle** [Esri/calcite-components#1886]
### Design
Light mode outline: rgba(0, 0, 0, .15)
![image](https://user-images.githubusercontent.com/14083530/113323550-b3627780-92db-11eb-9ea0-a08d37066edc.png)

Dark mode outline: rgba(255, 255, 255, .15)
![image](https://user-images.githubusercontent.com/14083530/113323488-a04fa780-92db-11eb-894a-45f6ae9ec041.png)

### Current build
(colors close to the background get lost):

![image](https://user-images.githubusercontent.com/14083530/113322899-e22c1e00-92da-11eb-8fc2-cb8ba57cf698.png)

![image](https://user-images.githubusercontent.com/14083530/113322857-d3de0200-92da-11eb-97c1-72ee6462633a.png)

**Comment by julio8a** [comment]
This should be addressed in the refactor: #1679

**Comment by jcfranco** [comment]
Installed.

**Comment by jcfranco** [comment]
Closing as self-verified.
